content/en/docs/introduction/what's-new-in-3.2.0.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "What's New in 3.2.0"
+keywords: 'Kubernetes, KubeSphere, new features'
+description: "What's New in 3.2.0"
+linkTitle: "What's New in 3.2.0"
+weight: 1400
+---
+
+On Nov 2, 2021, GPU resource scheduling and management and GPU usage monitoring further improve user experience in cloud-native AI scenarios. Moreover, enhanced features such as multi-cluster management, multi-tenant management , observability, DevOps, app store, and service mesh further perfect the interactive design for better user experience.
+
+In addition to the above highlights, KubeSphere 3.2.0 also features other functionality upgrades and fixed the known bugs. There were some deprecated or removed features in 3.2.0. For more and detailed information, see [Release Notes for 3.2.0](../../release/release-v320/).
```

Comment:
Do we have any `deprecated or removed features` in 3.2.0? In the release note for 3.2.0, only an API was mentioned as deprecated.